analytics/ga.go

```diff
@@ -60,6 +60,7 @@ func NewUserSession() *UserSession {
 
 // ScreenView is the public function to post a ScreenView analytics message to GA
 func ScreenView(screen string) {
+	logging.Stderr("[Analytics] Gathering analytics for screen-view, %s", screen)
```

Comment:
perhaps rephrase screen-view, user probably won't know about Google Analytics? This is just initialising the GA session, so perhaps something like that